src/components/TimeRangePicker/index.tsx

```diff
@@ -55,9 +55,22 @@ const defaultProps = {
       unit: "months",
       default: 3,
     },
+    {
```

Comment:
E-flux should have their own sets of timeOptions, which doesnt have to same for other projects. 
So this should not be added here. 

Note the timeOptions can be overwritten as they are just defaultProps

Reply:
Ah I see they can be overwritten. But for the storyBook demo it it not useful to show all options initially?

Reply:
I will fix, i will trim some of the default options while i am at, to force people to implement their defaults (if they want something special) As ideally there shouldn't be scroll.

Reply:
> I will fix, i will trim some of the default options while i am at, to force people to implement their defaults (if they want something special) As ideally there shouldn't be scroll.

Okay cool! Does anything still needs to be done before this is merged or is it ready?